matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/auth/AuthAPI.kt

```diff
@@ -68,6 +69,14 @@ internal interface AuthAPI {
     @POST(NetworkConstants.URI_API_PREFIX_PATH_R0 + "register")
     suspend fun register(@Body registrationParams: RegistrationParams): Credentials
 
+    /**
+     * Register to the homeserver, or get error 401 with a RegistrationFlowResponse object if registration is incomplete
+     * method to perform other custom stages
+     * Ref: https://matrix.org/docs/spec/client_server/latest#account-registration-and-management
+     */
+    @POST(NetworkConstants.URI_API_PREFIX_PATH_R0 + "register")
+    suspend fun registerOther(@Body registrationOtherParams: RegistrationOtherParams): Credentials
```

Comment:
Can you rename `Other` to `Custom` please (so 3 times here)?

Reply:
done